build/android/generate_xwalk_core_library.py

```diff
@@ -143,28 +151,13 @@ def CopyBinaries(out_dir, no_icu_data, use_lzma):
   pak_list_xml.writexml(pak_list_file, newl='\n', encoding='utf-8')
   pak_list_file.close()
 
-  # Copy jar files to libs.
-  libs_dir = os.path.join(out_dir, LIBRARY_PROJECT_NAME, 'libs')
-  if not os.path.exists(libs_dir):
-    os.mkdir(libs_dir)
-
-  libs_to_copy = [
-      'xwalk_core_library_java_app_part.jar',
-      'xwalk_core_library_java_library_part.jar',
-  ]
-
-  for lib in libs_to_copy:
-    source_file = os.path.join(out_dir, 'lib.java', lib)
-    target_file = os.path.join(libs_dir, lib)
-    shutil.copyfile(source_file, target_file)
-
   # Copy native libraries.
   source_dir = os.path.join(out_dir, XWALK_CORE_SHELL_APK, 'libs')
   distutils.dir_util.copy_tree(source_dir, libs_dir)
 
   # NOTE: Gradle doesn't accept '-', use '_' instead.
   if use_lzma:
-    for arch in ['x86', 'armeabi_v7a']:
+    for arch in ['x86', 'armeabi-v7a']:
```

Comment:
Gradle will report error if we pack the library use armeabi-v7a suffix, please use armeabi_v7a instead.